packages/survey-creator-knockout/src/components/results-table-row.html

```diff
@@ -1,18 +1,21 @@
+<!-- ko with: model -->
 <tr data-bind="click: toggle, key2click">
-    <td class="svd-dark-border-color">
+    <td class="svd-dark-border-color"
+        data-bind="style: { paddingLeft: 'calc(' + (6 + 2 * lvl) + ' * 8px)' }">
```

Comment:
It is better to transfer this calculation to the model, and then the duplication will be removed.